PaymentHighway/Media.xcassets/ccnumLight.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "ccnumLight.png",
```

Comment:
Could use PDF vector icons instead of PNGs? This is a general comment applying to all of these asset changes.

Reply:
Thanks! i changed it now.